modules/ipi-install-removing-the-provisioner-node.adoc

```diff
@@ -0,0 +1,15 @@
+// This is included in the following assemblies:
+//
+// installing/installing_bare_metal_ipi/ipi-install-post-installation-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="ipi-install-removing-the-provisioner-node_{context}"]
+= Removing the provisioner node
+
+Optionally, the provisioner node can be removed after installation.
+
+.Procedure
+
+. Verify a backup of the `install-config.yaml` file in case the cluster must be redeployed.
```

Comment:
"Verify a backup" does sound off to me. I'd suggest "Verify that you have a backup..." or even just "Take a backup..."

Also just wanted to check - this procedure talks about removing the provisioner node, but all it seems to do is make sure that you have backups of resources, and I don't see any steps to actually remove something. Is this correct as is?